res/css/views/rooms/_EventTile.scss

```diff
@@ -128,7 +145,6 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
         display: inline-block;
         padding-bottom: 0px;
         padding-top: 0px;
-        margin: 0px;
```

Comment:
- For modern layout: specified with https://github.com/matrix-org/matrix-react-sdk/pull/8711/files#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919R71
- For IRC layout: specified with https://github.com/matrix-org/matrix-react-sdk/pull/8711/files#diff-29c6a3cc88dee2f8be7fa065c46d7d64f66ba006cd8ba9f3f16f301f41d67444R176
- For bubble layout: inline start margin is not required